.github/workflows/codeql.yml

```diff
@@ -0,0 +1,68 @@
+name: "CodeQL Advanced"
+
+on:
+  push:
+    branches: [release, beta]
+  pull_request:
+    branches: [beta]
+    paths-ignore:
+    - '**.md'
+  schedule:
+    - cron: '43 2 * * 5'
+
+jobs:
+  analyze:
+    name: Analyze (${{ matrix.language }})
+    runs-on: ${{ (matrix.language == 'c-cpp' && 'windows-latest') || 'ubuntu-latest' }}
```

Comment:
Things look good to me but what's happening here?

only run on windows if language is c-cpp, otherwise only ubuntu? Do we have to consider mac or anything?

Reply:
This was leftover from the matrix of different languages to scan, removed - in early testing there was some error suggesting the C++ needed to be built on Windows, but it seems to be running fine on Ubuntu. Shouldn't matter which is used for scanning.